e2e/pages/session-search.po.ts

```diff
@@ -19,14 +20,17 @@ export class SessionSearchPage {
     private listingRequestsTablePaginator = new Paginator(element(by.id('hearings-part-table-paginator')));
     private listHearingButton = element(by.id('listHearingButton'));
     public assignButton = element(by.id('assign'));
+    public startTime = element(by.id('startTime'));
```

Comment:
That startTime element is a part of the SessionSearchPage or rather the dialog that appears after clicking 'List hearing'? If the second option is true, then I'd create a separate component for that dialog. In this case it looks like the SessionSearch page has some input/button/property called 'startTime' which is not true :(

Reply:
So the reason I added this here is because of 'session-e2e-spec.ts::124'.

This test opens the dialog on the calendar and checks it for the 'session.type' and 'hearing.caseTitle' etc.
I figured this would've been a good place to simply confirm the 'hearing.startTime' aswell.

To do this, I needed the previous test (session-e2e-spec.ts::115) to input a 'startTime'.
Doing this takes me to the function on 'session-search.po.ts::31'.

I understand that you're saying the 'SessionSearch' page does not have a button or input called 'startTime' but the 'listHearingButton' is present in this script, which is on the same dialog as the 'startTime' input.

I can maybe rename the variable to make it a little more clear?

Reply:
Yeah, let's just signify that this startTime element is on the dialog rather than the page (y)

Reply:
Done